src/script/components/MessagesList/Message/ContentMessage/ContentMessage.styles.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {CSSObject} from '@emotion/react';
+
+export const messageBodyWrapper: CSSObject = {
+  display: 'grid',
+  gridTemplateColumns: 'calc(100% - 56px) 56px',
+  paddingLeft: 'var(--conversation-message-sender-width)',
+};
+
+export const deliveredMessageIndicator: CSSObject = {
+  display: 'flex',
+  justifyContent: 'center',
+  paddingTop: '2px',
+  width: '56px',
```

Comment:
I think we could make this `56` a variable (a js variable is fine I guess). It appears 3 times in this file ;)